editquality/utilities/generate_make.py

```diff
@@ -4,20 +4,26 @@
 :Usage:
     generate_make -h | --help
     generate_make
-                 [--config-dir=<path>]
-                 [--diff]
-                 [--diff-only=<wiki_name>]
-                 [--output-path=<path>]
-                 [--template-dir=<path>]
+                 [--config=<path>]
+                 [--compare-to=<path>]
+                 [--compare-section=<name>]...
+                 [--output=<path>]
+                 [--templates=<path>]
+                 [--debug]
 
 :Options:
-    --config-dir=<path>     Directory to search for configuration files.
-    --diff                  Output diffs while generating.
-    --diff-only=<wiki_name> Only compare sections, don't generate.  Wiki name
-                            is given in English, for example "Spanish Wikibooks".
+    --config=<path>         Directory to search for configuration files
+                            [default: config/]
+    --compare-to=<path>     If specified, compare output to an old Makefile
+    --compare-section=<name> Only compare sections, don't generate.  Wiki name
+                             is given in English, for example
+                             "Spanish Wikibooks"
                             Multiple wikis can be separated by a comma.
-    --output-path=<path>    Where to write the Makefile output.
-    --template-dir=<path>   Directory to search for input templates.
+    --output=<path>         Where to write the Makefile output.
+                            [default: <stdout>]
```

Comment:
Stdout isn't my preference for this tool, but we discussed in IRC.  This is fine until we need to write multiple files...